src/brfss/01_brfss_data_profiling.Rmd

```diff
@@ -0,0 +1,248 @@
+---
+title: "BRFSS Data Profiling"
+author: "Crystal"
+output: 
+  rmarkdown::html_document:
+    toc: true
+    toc_float: true
+    highlight: tango
+    theme: journal
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+#library(SASxport)
+library(tidyverse)
+#library(survey)
+library(foreign)
+library(DataExplorer)
+library(ggformula)
+library(mosaic)
+library(manipulate)
+library(RColorBrewer)
+library(gt)
+library(readr)
+```
+
+
+Reference codebook: https://www.cdc.gov/brfss/annual_data/2018/pdf/codebook18_llcp-v2-508.pdf
+```{r message=FALSE, warning=FALSE, paged.print=FALSE}
+brfss2018 <- read_csv("~/git/dspg20career/src/brfss/brfss2018.csv")%>%
+  select(-X1)
+```
+
+```{r eval=FALSE, message=FALSE, warning=FALSE, include=FALSE, paged.print=FALSE}
+brfss2018_subset <- brfss2018 %>%
+  select(
+    X_STATE,
+    X_AGE80, #Imputed Age value collapsed above 80
+    X_RACE, #cleaned
+  #  X_BMI5CAT,
+    X_INCOMG, #cleaned
+    SEX1, #cleaned
+    #MARITAL,
+    EDUCA, #cleaned
+   # RENTHOM1,
+    VETERAN3, #cleaned
+    EMPLOY1,  #cleaned
+    INCOME2 #cleaned
+  ) %>%
+  mutate(race = if_else(
```

Comment:
I think you can use recode for mapping those values (https://dplyr.tidyverse.org/reference/recode.html)